acceptance-tests/profiler-stress.mk

```diff
@@ -0,0 +1,12 @@
+SYS_REFS = \
+	System \
+	System.Core \
+	System.Data \
+	System.Runtime.Serialization \
+	System.Xml \
+	System.Xml.Linq
```

Comment:
roslyn needs it, mcs doesn't, and we have been switching everything over to be consistent.